docs/virtual-devices.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Virtual Devices
+
+contributors: [nakata5321]
+---
+
+**This article will tell you how to create virtual devices in a smart home, so you can see what the actual platform looks like.**
+
+## Install integration
+
+To use virtual devices you need to install ["demo" integration](https://www.home-assistant.io/integrations/demo/). 
+To do this, you should add next string to your configuration file:
+
+Go to configuration folder, which you provide due configuration process. 
```

Comment:
Go to the configuration folder, which you provided during the configuration process. In this folder, you will find a folder named "homeassistant". Go inside it. Open the `configuration.yaml` file with a text editor under the **root** user and insert the following line into it: